python/shotgun_model/simple_shotgun_hierarchy_model.py

```diff
@@ -31,7 +31,7 @@ class SimpleShotgunHierarchyModel(ShotgunHierarchyModel):
     can also be subclassed from this class.
     """
 
-    def load_data(self, seed_entity_field, path=None, entity_fields=None):
+    def load_data(self, seed_entity_field, root=None, entity_fields=None):
```

Comment:
@josh-t your call, you know this better than i do. I think either we do 100% backwards compatibility or drop it completely. no middle ground solution.